Classes/Cache/Frontend/VariableFrontendWithWarmupReservation.php

```diff
@@ -59,4 +71,14 @@ public function flushByTags(array $tags)
         $queueService = GeneralUtility::makeInstance(QueueService::class);
         $queueService->queueMany($urls);
     }
+    
+    private function areAllGetParamsAllowed($whitelistedGetParams, $getParams): bool
```

Comment:
please provide type hints for the function parameters

Reply:
fixed